### ISSUE TITLE
An error message display when this page is showing

### ISSUE THREAD

**Issue by andycnzh** [OfficeDev/Office-IT-Pro-Deployment-Scripts#325]
When I open this page, I see an dialog just show "Error", no detail info.

Then I found I can't choose any list for the Version dropdown list, the three dropdown list just show blank.

Does anyone see the same issue?

**Comment by tnunberger** [comment]
Yes, I have the same problem accross multiple browsers. I hope it doesn't but seems like the editor retires :(. I would much appreciate if the error could be fixed.

**Comment by ItzLevvie** [comment]
@andycnzh Please use https://config.office.com

The development of https://github.com/OfficeDev/Office-IT-Pro-Deployment-Scripts has stopped.
